testing/unit/test_copy.py

```diff
@@ -1,40 +1,158 @@
 import os
+import types
+import unittest
 
 from nanome.api import structure as struct
-from testing.unit.utilities import alter_object, assert_equal, create_full_tree, TestOptions
+from nanome import util
 
-import unittest
 
-test_assets = os.getcwd() + ("/testing/test_assets")
-options = TestOptions(ignore_vars=["_unique_identifier", "_index", "_alt_loc"])
+test_assets = os.path.join(os.getcwd(), 'testing', 'test_assets')
 
 
-def shallow_copy_tester(constructor):
-    original = constructor()
-    alter_object(original)
-    copy = original._shallow_copy()
-    assert_equal(original, copy, options)
+class CopyTestCase(unittest.TestCase):
+    """Validate Structure shallow and deep copy functions."""
 
+    @classmethod
+    def setUpClass(cls):
+        pdb_path = os.path.join(test_assets, 'pdb', '3mcf.pdb')
```

Comment:
should also test with a multi-frame small molecule, maybe the thrombin test asset?

Reply:
good call